tox.ini

```diff
@@ -35,6 +35,8 @@ commands =
     format: isort --check .
     tests: pip install . -r tests/requirements.txt -c tests/constraints.txt
     tests: pytest
+    alltests: pip install . -r tests/requirements.txt -c tests/constraints.txt
```

Comment:
I don't know why trieste uses both `tox` and `poetry` commands, but should there be commands here to run the "slow tests" and the "quick tests"? It looks like the other `poetry` commands are reproduced here.

Reply:
We're not using poetry at all actually, just taskipy. The taskipy commands are merely intended to make it easier to run things locally, flexibly and quickly, while the tox commands are intended to be more reliable (in that they generate the venv each time) and are what's used by the build checks.